client/src/app/site/site.component.ts

```diff
@@ -83,6 +105,18 @@ export class SiteComponent extends BaseComponent implements OnInit {
         });
 
         this.searchform = new FormGroup({ query: new FormControl([]) });
+
```

Comment:
All changes here can be moved into the service I guess. This does not excatly has anything todo with the site component. The check is independend from the component, becuase it's activated/deactivated through `noInterruption`.

Reply:
- you cannot simply get route-data in a service. `this.router.events` Is what you really should have
- triggering UI-events from a global serviceMethods from a service should be avoided, therefore: no snackbar in this service.
- logically, update notifications and interruptions are required in the site-container. Therefore, `site.ts` is the correct file
- projectors opened in a new tab will *never* subscribe to this, because they have never visited the site-component